### PR TITLE
Upgrade ckb sdk version

### PR DESCRIPTION
**Pull request by cezres**
Use the sdk develop branch
Because the test node use the ckb develop branch

### CODE REVIEW

File: faucet-server/Package.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/vapor/vapor.git", from: "3.0.0"),
         .package(url: "https://github.com/cezres/SQLite.swift.git", from: "0.11.6"),
-        .package(url: "https://github.com/cezres/ckb-sdk-swift", .revision("e8d2205fa2105f3855a6c535ab3a679c35b147b3")),
+        .package(url: "https://github.com/cezres/ckb-sdk-swift", .revision("df1c1d0bacdda8550d84f343121822b9c480dcf2")),
```
Comment (ashchan) [inline faucet-server/Package.swift]: `develop` already merged this commit.

File: faucet-server/Sources/App/Services/Wallet/Wallet.swift [hunk 2]
```diff
@@ -17,15 +17,21 @@ public class Wallet {
...
-        return Script.verifyScript(for: publicKey, codeHash: systemScriptCellHash)
-    }
-    var lockHash: H256 {
+    }()
+    lazy var address: String = {
+        return AddressGenerator(network: .testnet).address(for: publicKey)
+    }()
+    lazy var publicKeyHash: String = {
+        return "0x" + AddressGenerator(network: .testnet).publicKeyHash(for: address)!
+    }()
```
Comment (ashchan) [inline faucet-server/Sources/App/Services/Wallet/Wallet.swift]: Are these computing properties (`publicKey`, `address` and `publicKeyHash`) really very slow to access that they have to be lazy ones?

Comment (cezres) [reply faucet-server/Sources/App/Services/Wallet/Wallet.swift]: It only takes about 3ms in my device, should not need to be  lazy ones 
but I don't think they need to be executed repeatedly.

Comment (cezres) [reply faucet-server/Sources/App/Services/Wallet/Wallet.swift]: Ok

**Pull request merged**